dandi/organize.py

```diff
@@ -1025,3 +1028,75 @@ def msg_(msg, n, cond=None):
         msg_(" %d invalid not considered.", skip_invalid),
         dandiset_path.rstrip("/"),
     )
+
+
+LABELREGEX = r"[^_*\\/<>:|\"'?%@;.]+"
+ORGANIZED_FILENAME_REGEX = (
+    rf"sub-{LABELREGEX}(_(ses|tis|slice|cell)-{LABELREGEX})*(_[a-z]+(\+[a-z]+)*)?\.nwb"
```

Comment:
- let's make at least `_ses` strictly following `sub-` if present
- according to https://github.com/dandi/dandi-cli/blob/HEAD/dandi/organize.py#L45 is still missing `probe` and `obj`.

Reply:
ideally there should be a single data structure describing entities etc we should have in the filename. In principle that `potential_fields` is the one but I guess can't be used here "as is", unless you enrich that structure with regexes for the values, in particular for extensions etc.  Then this regex  can be constructed programmatically and we would avoid duplication of this knowledge.

Reply:
@yarikoptic Can tis/slice/cell/probe/obj occur without a preceding ses, or is ses a prerequisite for them?

Reply:
according to `potential_fields`, `ses` is not mandatory, so other entities could occur without a preceding `ses`.

Reply:
Done: 55df999.